src/main/java/pl/javastart/task/Main.java

```diff
@@ -3,6 +3,28 @@
 public class Main {
 
     public static void main(String[] args) {
+        Microphone mic1 = new Microphone("Behringer", "C-1U", 225,"condenser" , 44.1,
+                48, 16, false);
+        Microphone mic2 = new Microphone("Razer", "Seiren-X", 449, "condenser", 20,
+                40, 16, false);
 
+
+        System.out.println("Microphone 1:\n" +
+                "Model: " + mic1.producer + " " + mic1.model + "\n" +
+                "Price: " + mic1.price + "zł\n" +
+                "Sample rate from " + mic1.minSampleRate + "kHz to " + mic1.maxSampleRate + "kHz\n" +
+                "Bitrate: " + mic1.bitRate + "bit\n" +
+                "Transducer type: " + mic1.transducerType + "\n" +
+                "Is it wireless: " + mic1. isWireless
+                );
```

Comment:
Checkstyle zauważył w tej linii błąd. To, co znalazł możesz podejrzeć w pull requeście z zakładce "Files changed". Polecam zainstalować sobie dodatek do intellij, żeby na bieżąco informował. Instrukcję znajdziesz tutaj https://bootcamp.javastart.pl/strona/jakosc-kodu